src/_locales/en/messages.json

```
@@ -89,8 +89,17 @@
   "welcomePageAlwaysPrivateCheckbox": {
     "message": "Always open sites in a Firefox Private Window when switching"
   },
-  "welcomePageTelemetryCheckbox": {
-    "message": "Allow Mozilla to collect information about how you use this extension so we can make it better. <a>Privacy Notice</a>"
+  "welcomePageTelemetryCheckboxNotice": {
+    "message": "Allow Mozilla to collect information about how you use this extension so we can make it better."
+  },
+  "welcomePagePrivacyPolicyIntro": {
+    "message": "The Firefox Bridge extension allows you to open a web page in the browser of your choice. The <a>Mozilla Privacy Policy</a> describes how we handle information that we receive from you."
+  },
+  "welcomePagePrivacyPolicyDescription": {
+    "message": "Here are the key things you should know about how the Firefox Bridge Extension handles your data:"
```

```suggestion
    "message": "Here are the key things you should know about how the Firefox Bridge extension handles your data:"
```
"Extension" here, but "extension" in the previous string. Which one is correct? I assume extension, given David's comment.